src/main/java/eu/europeana/thumbnail/web/ThumbnailController.java

```diff
@@ -280,5 +324,19 @@ private String computeResourceUrl(final String resourceUrl, final String resourc
         return getMD5(resourceUrl) + "-" + (StringUtils.equalsIgnoreCase(resourceSize, "w200") ? "MEDIUM" : "LARGE");
     }
 
+    private ObjectMetadata getMetaData(String id)
+    {
```

Comment:
Our code convention is to put the open curly bracket on the same line as the method definition